tools/create_service_bundle.py

```diff
@@ -0,0 +1,568 @@
+            self.service_name,
+            "pod status --json",
+            print_output=False,
+        )
+
+        self.write_file("service_pod_status.json", output)
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def create_plan_status_file(self, plan):
+        output = sdk_cmd.svc_cli(
+            self.package_name,
+            self.service_name,
+            "plan status {} --json".format(plan),
+            print_output=False,
+        )
+
+        self.write_file("service_plan_status_{}.json".format(plan), output)
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def create_plans_status_files(self):
+        plans = sdk_cmd.svc_cli(
+            self.package_name,
+            self.service_name,
+            "plan list",
+            json=True,
+            print_output=False,
+        )
+
+        for plan in plans:
+            self.create_plan_status_file(plan)
+
+    def create_log_files(self):
+        scheduler_tasks = get_scheduler_tasks(marathon_app_show(self.service_name))
+        pod_tasks = get_tasks(self.framework_id)["tasks"]
+        all_tasks = scheduler_tasks + pod_tasks
+        tasks_by_agent_id = dict(groupby("slave_id", all_tasks))
+        agent_id_by_task_ids = dict(
+            map(lambda task: (task["id"], task["slave_id"]), all_tasks)
+        )
+
+        agent_executor_paths = {}
+        for agent_id in tasks_by_agent_id.keys():
+            agent_executor_paths[agent_id] = debug_agent_files(agent_id)
+
+        task_executor_sandbox_paths = {}
+        for agent_id, tasks in tasks_by_agent_id.items():
+            for task in tasks:
+                task_executor_sandbox_paths[
+                    task["id"]
+                ] = sdk_diag._find_matching_executor_path(
+                    agent_executor_paths[agent_id], sdk_diag._TaskEntry(task)
+                )
+
+        for task_id, task_executor_sandbox_path in task_executor_sandbox_paths.items():
+            agent_id = agent_id_by_task_ids[task_id]
+            download_task_files(
+                agent_id,
+                task_executor_sandbox_path,
+                task_id,
+                os.path.join(self.directory_name, "tasks"),
+                self.DOWNLOAD_FILES_WITH_PATTERNS,
+            )
+
+    def create(self):
+        self.install_service_cli()
+        self.create_dcos_version_file()
+        self.create_task_file()
+        self.create_marathon_task_list_file()
+        self.create_configuration_file()
+        self.create_pod_status_file()
+        self.create_plans_status_files()
+        self.create_log_files()
+        BASE_TECH_BUNDLE[self.package_name](
+            self.package_name, self.service_name, self.directory_name
+        ).create()
+
+
+def print_usage(argv):
+    logger.info("TODO: usage")
+
+
+def main(argv):
+    if len(argv) < 2:
```

Comment:
Not critical now, but could we use `argparse`? Then we would get `usage` for free.

Reply:
Done.